test/SmallTests/test_small.py

```diff
@@ -0,0 +1,50 @@
+import sys
+sys.path.append('../../')
+from ProjectLib import XMLtoProject
+from TimeLoopLib import TreeDynamicTimeStepping
+import unittest
+import glob
+import os
+
+filepathExampleSetups = "./Test_Setups_small/*.xml"
+xml = glob.glob(filepathExampleSetups)
+e = []
+for xmlfile in xml:
+    resultsFilepath = "./Test_Setups_small/testoutputs/*.*"
+    results = glob.glob(resultsFilepath)
+    for result in results:
+        os.remove(result)
+    print("________________________________________________")
+    print("In the following the setup", xmlfile, "is tested.")
+
+    class MyTest(unittest.TestCase):
+
+        filepathExampleSetups = "./ProjectLib/ExampleSetups/*.xml"
```

Comment:
This fails, if the file is run from another path.
```suggestion
        filepathExampleSetups = path.join(path.dirname(path.abspath(__file__)),"Test_Setups_small/*.xml")
```